metrics/prometheus/metrics.go

```diff
@@ -0,0 +1,30 @@
+package prometheus
+
+// Metrics definitions
+// Counter      *prometheus.CounterVec
+// ResponseTime *prometheus.HistogramVec
+type Metrics struct {
+	constLabels map[string]string
+}
+// Method for creation new custom prometheus  metrics
+// pm := &Metrics{
+//		Counter: prometheus.NewCounterVec(
+//			prometheus.CounterOpts{
+//				Name:        "metrics_name",
+//				Help:        "Metrics Description",
+//				ConstLabels: constLabels,
+//			},
+//			[]string{"endpoint"},
+//		),
+//	}
+//	prometheus.Register(pm.Counter)
+func NewMetrics(version, hash, buildTime string) *Metrics {
+	pm := &Metrics{
+		constLabels: map[string]string{
+			"version":   version,
+			"hash":      hash,
+			"buildTime": buildTime,
+		},
+	}
+	return pm
```

Comment:
I propose to leave a empty middleware for prometheus with examples. It is given ability to simple add custom metrics if needed

Reply:
Yeah, that makes sense!